### PR TITLE
tweak script to use keys instead of roles

### PR DESCRIPTION
**Pull request by estute**
@JZ switched script to use aws keys following our conversation this morning. Also, set the jobs produced by this to run at different times. Also, I simply cat out the script output, since the virtualenv plugin is hardwired to use dash, not bash, and therefore no pipefail options.

### CODE REVIEW

**Comment by benpatterson** [comment]
I think you want @jzoldak

**Comment by estute** [comment]
@benpatterson as well

**Comment by benpatterson** [comment]
LGTM

File: testeng/jobs/backupJenkins.groovy [hunk 1]
```diff
@@ -82,19 +92,22 @@ secretMap.each { jobConfigs ->
...
-        script += "sync\n"
+        String script = "sync\n"
         // This might seem overkill, but in case the pip requirements change, read them from
         // the requirements file in the workspace
         readFileFromWorkspace('testeng/resources/requirements.txt').split("\n").each { line ->
             script += "pip install --exists-action w ${line}\n"
         }
-        script += "aws ec2 create-snapshot --volume-id ${jobConfig['volumeId']} --description 'Automatic ${jobConfig['jenkinsInstance']} jenkins snapshot' |tee \${WORKSPACE}/snapshot-out"
+        script += "aws ec2 create-snapshot --volume-id ${jobConfig['volumeId']} --description 'Automatic ${jobConfig['jenkinsInstance']} jenkins snapshot' > \${WORKSPACE}/snapshot-out\n"
+        script += "cat \${WORKSPACE}/snapshot-out"
```
Comment (jzoldak) [inline testeng/jobs/backupJenkins.groovy]: nit: this should have a file extension. I'd recommend .log.

**Pull request merged**